app/components/elements/forms/select_field_component.rb

```diff
@@ -3,32 +3,14 @@
 module Elements
   module Forms
     # Component for form select fields
-    class SelectFieldComponent < ApplicationComponent
-      def initialize(form:, field_name:, options:, required: false, hidden_label: false, # rubocop:disable Metrics/ParameterLists
-                     label: nil, help_text: nil, prompt: false, data: {})
-        @form = form
-        @field_name = field_name
+    class SelectFieldComponent < FieldComponent
+      def initialize(options:, prompt: nil, **args)
```

Comment:
Could we use anonymous forwarding (`...`) instead of `**args` on lines 7 and 10?